src/app/public/modules/action-hub/action-hub.component.html

```diff
@@ -0,0 +1,41 @@
+<sky-wait [isFullPage]="true" [isWaiting]="loading"></sky-wait>
+<sky-fluid-grid *ngIf="!loading">
+  <sky-row>
+    <sky-column>
+      <h1 class="sky-font-heading-1">{{ title }}</h1>
+    </sky-column>
+  </sky-row>
+  <sky-row>
+    <sky-column screenLarge="9" screenXSmall="12">
+      <div
+        [skyThemeClass]="{
+          'sky-margin-stacked-separate': 'default',
+          'sky-margin-stacked-xl': 'modern'
+        }"
+      >
+        <ng-content select="sky-action-hub-buttons"></ng-content>
+      </div>
+
+      <div
+        [skyThemeClass]="{
+          'sky-margin-stacked-separate': 'default',
+          'sky-margin-stacked-xxl': 'modern'
+        }"
+      >
+        <sky-needs-attention [items]="needsAttention"></sky-needs-attention>
+      </div>
+
+      <ng-content select="sky-action-hub-content"></ng-content>
```

Comment:
We should wrap the content area with `sky-margin-stacked-xxl` for modern and `sky-margin-stacked-separate` for default theme to provide spacing between the content and Related Links in responsive modes.